src/api/migrations/0004_auto_20180204_1307.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.2 on 2018-02-04 13:07
+from __future__ import unicode_literals
+
+import api.models
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0003_auto_20170824_0821'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='comparefileupload',
+            name='file1',
+            field=models.FileField(upload_to=api.models.user_directory_path),
+        ),
+        migrations.AlterField(
+            model_name='comparefileupload',
+            name='file2',
+            field=models.FileField(upload_to=api.models.user_directory_path),
+        ),
+        migrations.AlterField(
+            model_name='convertfileupload',
+            name='file',
+            field=models.FileField(upload_to=api.models.user_directory_path),
+        ),
+        migrations.AlterField(
+            model_name='validatefileupload',
+            name='file',
+            field=models.FileField(upload_to=api.models.user_directory_path),
+        ),
+    ]
```

Comment:
Don't commit these local test migrations, since there are no database changes. Just file `0001_initial.py` and `__init__.py` from the master is enough.